spec/concurrent/edge/future_spec.rb

```diff
@@ -313,6 +313,14 @@
       f = Concurrent.future { Concurrent.future { 1 } }.flat.then(&:succ)
       expect(f.value!).to eq 2
     end
+
+    it 'completes future when Exception raised' do
+      f = Concurrent.future { raise Exception, 'fail' }
+      sleep 0.2
```

Comment:
Please avoid sleep in tests, they cause timing issues and slow down tests. It can be replaced with `f.wait(1)`. (I'll fix it in follow up PR)